frontend/src/pages/category/Category.js

```diff
@@ -0,0 +1,146 @@
+import {useContext, useEffect, useState} from "react";
+import {AuthContext} from "../../context/authContext";
+import Page from "../../components/Page";
+import {FaEdit} from "react-icons/fa";
+import React from "react";
+import ErrorMessage from "../../messages/errorMessage";
+import ModalAddCategory from "./ModalAddCategory";
+import ModalEditCategory from "./ModalEditCategory";
+import Table from "react-bootstrap/Table";
+import Button from "react-bootstrap/Button";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import TogglePage from "../../components/TogglePage";
+import CardContainer from "../../components/CardContainer";
+
+export default () => {
+
+    const [currentCustomerId, setSelected] = useState(JSON.parse(localStorage.getItem('user')).customers[0].id);
```

Comment:
What is user is not in the local storage of user doesn't have customers?

Reply:
changed everywhere to 
    const currentCustomerId = localStorage.
    getItem("currentCustomerId") != null ? localStorage.
    getItem("currentCustomerId"): 0;

Reply:
done